documentation/asciidoc/topics/ref_persistent_cache_store.adoc

```diff
@@ -0,0 +1,14 @@
+[id='persistent-volume-claims_{context}']
+= Persistent volume claims
+{ispn_operator} creates a persistent volume claim (PVC) and mounts container storage at: +
+`{pv_path}`
+
+.Caches
+When you create caches, {brandname} permanently stores them so they are available after cluster restarts.
```

Comment:
"permanently stores their configuration"? The current wording might be missinterpreted as the entire cache, including it's data, being permanently stored.